packages/lit-dev-content/site/tutorials/content/first-component/04.md

```diff
@@ -36,6 +37,16 @@ Define a method `tickToNextWord` that will increment `this.idx`.
 Calling this method increments our reactive internal state `idx` which will
 schedule an update.
 
+<litdev-aside type="info" noheading>
+
+When adding listeners imperatively with `addEventListener`, an arrow function is
```

Comment:
In this case, it's not specifically `addEventListener` but the callback to `setInterval`. Maybe this should be sort of basic knowledge for JS users but maybe warrants a link to https://developer.mozilla.org/en-US/docs/Web/API/setInterval#the_this_problem

Reply:
*facepalm* - Thank you for catching this twice. I was completely anchored on events.